tooling/nargo_cli/src/cli/check_cmd.rs

```diff
@@ -69,7 +81,8 @@ fn check_package(
     parsed_files: &ParsedFiles,
     package: &Package,
     compile_options: &CompileOptions,
-) -> Result<(), CompileError> {
+    allow_override: bool,
+) -> Result<bool, CompileError> {
```

Comment:
Can you add a doc comment above `check_package` explaining what the returned bool indicates? I can see from the function body, but can't tell from the signature alone.